package.json

```diff
@@ -20,5 +20,12 @@
   "homepage": "https://github.com/CodersCampCrew/Jobiz#readme",
   "devDependencies": {
     "snowpack": "^3.8.8"
-  }
+  },
+  "dependencies": {
+    "config": "^3.3.6",
+    "node-fetch": "^3.1.0",
+    "xhr": "^2.6.0",
+    "xhr2": "^0.2.1"
```

Comment:
We don't use those packages. We can remove them safely.
Instead of using `node-fetch` we can just use native Fetch API because we run website in a browser not in the node.